pkg/util/devices.go

```diff
@@ -135,6 +135,17 @@ func IsDeviceDynamicDirectPathIO(dev vimTypes.BaseVirtualDevice) bool {
 	return false
 }
 
+// SelectVGPU returns a slice of vGPU devices.
```

Comment:
@sreyasn Please file a GH issue to track updating this function name to `SelectVGPUWithVMIOPBacking`.

Reply:
can vGPUs have other Backing types? I thought they could only have VMIOP?

Reply:
nVidia GPUs use VMIOP. Intel GPUs use PCI Passthrough with DVX backings. If we wanted to rename the function to `SelectNVidiaVGPU`, that would also be fine, but I did not want to be so specific.